compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -378,8 +388,7 @@ fn maybe_use_metavar_location(
         })
     );
     if undelimited_seq {
-        // Do not record metavar spans for tokens from undelimited sequences, for perf reasons.
-        return orig_tt.clone();
+        // Do not record metavar spans for tokens from undelimited sequences, for perf reasons.        return orig_tt.clone();
```

Comment:
At least some tests on CI are failing due to this change.

Reply:
Thanks! I have fixed it